client/js/components/OnboardingIntro/styles.js

```diff
@@ -0,0 +1,25 @@
+import { StyleSheet, Dimensions } from "react-native";
+const { width, height } = Dimensions.get("window");
```

Comment:
`width` and `height` are keywords. Name them something else like screenWidth

Reply:
That's a good idea, Tommy.